### DISCUSSION TITLE
React flow for commercial use

### DISCUSSION THREAD

**Discussion by Rajgupta7080** [xyflow/xyflow#3369] [Q&A]
Can we use react flow normal feature in commercial projects for free or not?

**Comment by chrtze** [comment]
Yes, you can. React Flow is MIT licensed and can be used for free in any type of project.

**Reply by Rajgupta7080** [reply]
Then why did you written that on website
https://reactflow.dev/docs/guides/remove-attribution/
If you’re considering removing the attribution, we’d first like to mention:

If you’re using React Flow at your organization and making money from it, we rely on your support to keep React Flow developed and maintained under an MIT License. Before you remove the attribution, [see the ways you can support React Flow to keep it running](https://pro.reactflow.dev/pricing).

Are you using React Flow for a personal project? Great! Go ahead and remove the attribution. You can support us by reporting any bugs you find, sending us screenshots of your projects, and starring us on [Github](https://github.com/wbkd/react-flow). If you start making money using React Flow or use it in an organization in the future, we would ask that you re-add the attribution or sign up for one of our subscriptions.

Thank you for supporting the React Flow team ✌🏻

-Moritz, Christopher, and John

**Reply by hayleigh-dot-dev** [reply]
We ask folks that are making money of React Flow to consider supporting the project directly so we can continue to work on the library and keep the lights on. But It's **not** a requirement and we'll always provide React Flow under the MIT license which means any individual or organisation is basically* free to do whatever they want with the project.

If you want to support React Flow because you're building a commercial project off it you can, but it's also totally OK if you don't want to or can't. 😄 

*You can read the MIT license [here](https://github.com/wbkd/react-flow/blob/main/LICENSE)

**Reply by Rajgupta7080** [reply]
We already paid for the product and got a license too. We appreciate your work guys. Thank you for making this good library.
